tests/main/views/test_buyers.py

```diff
@@ -3676,3 +3642,122 @@ def test_award_brief_details_post_raises_400_and_displays_error_messages_and_pre
             assert document.xpath('//input[@id="input-awardedContractStartDate-day"]/@value')[0] == "x"
             assert document.xpath('//input[@id="input-awardedContractStartDate-month"]/@value')[0] == "y"
             assert document.xpath('//input[@id="input-awardedContractStartDate-year"]/@value')[0] == "z"
+
+
+class TestCancelBrief(BaseApplicationTest):
+    url = '/buyers/frameworks/digital-outcomes-and-specialists-2/requirements/digital-outcomes/{brief_id}/cancel'
+
+    def setup_method(self, method):
+        super(TestCancelBrief, self).setup_method(method)
+
+        self.data_api_client_patch = mock.patch('app.main.views.buyers.data_api_client', autospec=True)
+        self.data_api_client = self.data_api_client_patch.start()
+
+        self.data_api_client.get_framework.return_value = api_stubs.framework(
+            slug='digital-outcomes-and-specialists-2',
+            status='live',
+            lots=[
+                api_stubs.lot(slug='digital-outcomes', allows_brief=True),
+            ]
+        )
+        self.brief = api_stubs.brief(
+            user_id=123,
+            framework_slug='digital-outcomes-and-specialists-2',
+            lot_slug="digital-outcomes",
+            status='closed'
+        )['briefs']
+        self.brief["lotName"] = "Digital Outcomes"
```

Comment:
We definitely need to include this line?

Reply:
If the brief doesn't have a title attribute then it defaults back to lot name. The api stub doesn't have a 'title'.

However I can remove it it we do a `brief.get('lotName')` in the form instead of a `brief['lotName']`

Reply:
Cool, I don't mind either really. It was more it looked a little out of place.